py/bitbox02/bitbox02/bitbox02/bootloader.py

```diff
@@ -94,6 +104,17 @@ def versions(self) -> typing.Tuple[int, int]:
         firmware_v, signing_pubkeys_v = struct.unpack("<II", response[:8])
         return firmware_v, signing_pubkeys_v
 
+    def hardware(self) -> Hardware:
+        """
+        Returns (hardware variant).
+        """
+        # Previous bootloader versions does not support the call and have ATECC SC.
+        if self.version > "1.0.6":
```

Comment:
">= 1.1.0".

Reply:
I did this initially, but it wasn't working well with dev version, e.g. `v1.1.0-pre+48561ef480.dev >= 1.1.0`, seemed to return false in my tests.

Reply:
I checked again, it doesn't work for versions 1.1.0 with prerelease. I'd keep > 1.0.6, if you agree

Reply:
Check this out:

https://github.com/BitBoxSwiss/bitbox02-firmware/blob/c016621b126c61c9c94e16ec0cf799d25c042bbd/py/bitbox02/bitbox02/communication/bitbox_api_protocol.py#L557-L562

The `>1.0.6` way does not work consistently, so imho better to apply the above here as well.

Reply:
Ah, that's nice! But why `>1.0.6` doesn't work consistently? Anyway, your suggestion sounds better :+1:

Reply:
Fixed. I found a more readble/compact way to remove the pre-release `self.version.replace(prerelease=None)`. Should I update also the code you linked while I'm here?

Reply:
>  Ah, that's nice! But why >1.0.6 doesn't work consistently? Anyway, your suggestion sounds better 👍

1.0.7 could be released that patches something in 1.0.6 before 1.1.0 goes out, for example. Less likely in the bootloader, but did happen in the firmware a few times.

Reply:
@benma Good point, I didn't think about it. Anyway, I updated the code. Hopefully this could be the last round of review :pray: